frontend/containers/developer-about/component.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react';
+
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+import Link from 'next/link';
+
+import WebsiteSocialContact from 'containers/website-social-contact';
+
+import type { DeveloperAboutProps } from './types';
+
+export const DeveloperAbout: FC<DeveloperAboutProps> = ({
+  className,
+  developerId,
+  text,
+  developerName,
+  developerPhoto,
+  website,
+  social,
+  buttons,
+}: DeveloperAboutProps) => {
+  const intl = useIntl();
+
+  return (
+    <div
+      className={cx({
+        'flex flex-col gap-8 md:flex-row': true,
+        [className]: !!className,
+      })}
+    >
+      <div className="flex items-center md:items-start">
+        <div className="w-20 h-20">
+          <Image
```

Comment:
Can't you actually centre this image on mobile to make it less week?

Reply:
Can do. It actually looked more weird to me centred, because with the exception of a banner, all the content in the page will be left aligned.